### DISCUSSION TITLE
用editthiscookie提取的登陆错误，但是在CPP F12找不到cookies

### DISCUSSION THREAD

**Discussion by KMM5SS** [mikumifa/cppTickerBuy#26] [提问专区]
不好意思向您提问，本人并不是程序员，所以并不知道F12在哪提取cookies所以用了editthiscookies，但是一直错误，请问大佬怎么才能F12找到CPP的cookies?万分感谢了

**Comment by mikumifa** [comment]
cpp不用这个，是命令行登录
